### PR TITLE
Improve build system

### PR DESCRIPTION
**Pull request by NathanBnm**
Hi there! I figured out that your build system wasn't optimized, so I updated it. I moved some stuff into the `data` directory and created a loop to install icons. I made the necessary changes :wink:

**This PR is ready for review**

### CODE REVIEW

File: data/meson.build [hunk 1]
```diff
@@ -0,0 +1,39 @@
+#Install icons
+icon_sizes = ['16', '32', '48', '64', '128']
+
+foreach i : icon_sizes
+    install_data(
+        join_paths('icons', i, meson.project_name() + '.svg'),
```
Comment (bbuhler) [inline data/meson.build]: For some reason, I don't remember exactly, I needed to add the png icons as well.
I think the Polkit was not showing the svg icon.

Can you please add the png icons here to be save.

Comment (NathanBnm) [reply data/meson.build]: I'll take a look on this

Comment (NathanBnm) [reply data/meson.build]: I fixed this

**Comment by NathanBnm** [comment]
@bbuhler I just though maybe you have to update the icon cache by adding a `post_install.py` script, you currently don't do this.

It's simple you just have to create a folder named `meson`, then you create a file named `post_install.py` in there and you add the following code:

```python
#!/usr/bin/env python3

from os import path, environ
import subprocess

prefix = environ.get('MESON_INSTALL_PREFIX', '/usr/local')
schemadir = path.join(environ['MESON_INSTALL_PREFIX'], 'share', 'glib-2.0', 'schemas')
datadir = path.join(prefix, 'share')

if not environ.get('DESTDIR'):
    print('Compiling gsettings schemas…')
    subprocess.call(['glib-compile-schemas', schemadir])
    print('Updating icon cache…')
    subprocess.call(['gtk-update-icon-cache', '-qtf', path.join(datadir, 'icons', 'hicolor')])
```
Then you just have to add the following in at the end of you're main `meson.buil` file:

```meson
meson.add_install_script('meson/post_install.py') 
```

This will compile schemas and update icon cache. Maybe this is the right solution idk. Anyway this is a good thing to add this.

File: data/meson.build (hunk 1 above)
Comment (bbuhler) [reply data/meson.build]: You can even merge the two `install_data` calls.

```suggestion
        join_paths('icons', i, meson.project_name() + '.svg'),
        join_paths('icons', i, meson.project_name() + '.png'),
```

**Comment by bbuhler** [comment]
> @bbuhler I just though maybe you have to update the icon cache by adding a `post_install.py` script, you currently don't do this.

I take this in consideration next time I change the icons or when fixing #22.

**Comment by NathanBnm** [comment]
I didn't know that it was possible to merge the two `install_data` calls.

Great! ;)

**Pull request merged**